src/main/frontend/handler/editor.cljs

```diff
@@ -2511,10 +2512,10 @@
                                         (commands/handle-step [:editor/search-page]))}))]
         (state/set-editor-show-page-search! false)
         (let [selection (get-selection-and-format)
-              {:keys [selection-start selection-end value]} selection]
-          (if (not= selection-start selection-end)
+              {:keys [selection-start selection-end selection]} selection]
+          (if selection
             (do (delete-and-update input selection-start selection-end)
-                (insert (util/format "[[%s]]" value)))
+                (insert (util/format "[[%s]]" selection)))
```

Comment:
The function wrapped the whole block value, not the selection as page references.